scripts/promote_staging_to_prod.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+if [ ! -z $1 ]; then
+  checks=$(curl -s -H "Authorization: Bearer $1" "https://api.github.com/repos/chanzuckerberg/single-cell-data-portal/actions/runs?actor=czibuildbot" | jq -r '.workflow_runs[] | select(.display_title == "stage") | .conclusion' | head -n 1)
```

Comment:
what is `czibuildbot` exactly, and why are we including this as a query param? Does it narrow the workflows returned to only happy deploys (i.e. is this the only action run by that actor?)?

Reply:
I'm a little wary of using that as the selector. Can we make use of [this](https://docs.github.com/en/rest/actions/workflow-runs?apiVersion=2022-11-28#:~:text=List%20all%20workflow%20runs%20for%20a%20workflow.%20You%20can%20replace%20workflow_id%20with%20the%20workflow%20file%20name.%20For%20example%2C%20you%20could%20use%20main.yaml.%20You%20can%20use%20parameters%20to%20narrow%20the%20list%20of%20results.%20For%20more%20information%20about%20using%20parameters%2C%20see%20Parameters.) instead? Use the name of the happy deploy workflow file.

Reply:
totally missed that you could use the workflow file name instead of the ID, good find